package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Technology-Carbon-Standard",
+  "name": "GWF-Technology-Carbon-Standard",
```

Comment:
Not sure if this was changed manually? Probably need to regenerate it.

Reply:
That's come from the GWF fork. I'll revert that.